.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           - "aligner"
           - "annotation"
           - "default"
+          - "controlfreec"
           - "deepvariant"
           - "freebayes"
```

Comment:
should we keep freebayes in the CI?
Or should we try to fix it?

Reply:
yeah not sure, it does make local testing easier. Fixing I guess would either mean completely different test data or we could see if we can increase max time?

Reply:
Not sure we can go over 6 hours on GHA

Reply:
:/